client/view/Link/Link.css

```diff
@@ -0,0 +1,17 @@
+.Link {
+  padding-bottom: 0.2em;
+  margin-bottom: -0.2em;
+  color: var(--text-primary);
+  text-decoration: none;
+  background-image: linear-gradient(to top, currentcolor 1px, transparent 1px, transparent);
+  background-position: 0 1.3em;
+  transition: background-position var(--time);
+
+  --time: 0.3s;
```

Comment:
Why do we need a custom property here? Do we change it? Or do we use it somewhere else?